sql-plugin/src/main/scala/org/apache/spark/sql/rapids/RapidsShuffleInternalManagerBase.scala

```diff
@@ -1073,10 +1073,10 @@ abstract class RapidsShuffleInternalManagerBase(conf: SparkConf, val isDriver: B
   protected lazy val blockManager = env.blockManager
   protected lazy val shouldFallThroughOnEverything = {
     val fallThroughReasons = new ListBuffer[String]()
-    if (GpuShuffleEnv.isExternalShuffleEnabled) {
```

Comment:
minor nit, both statements could be put under one if statement:

```
if (!rapidsConf.isMultiThreadedShuffleManagerMode) {
  if (GpuShuffleEnv.isExternalShuffleEnabled) {
    fallThroughReasons += "External Shuffle Service is enabled"
  }
  if (GpuShuffleEnv.isSparkAuthenticateEnabled) {
    fallThroughReasons += "Spark authentication is enabled"
  }
}
```

Reply:
Agree!